src/util/positioning.ts

```diff
@@ -89,6 +93,42 @@ export class Positioning {
     return elOffset;
   }
 
+  private _getScrolledParent(element, includeHidden = false) {
```

Comment:
this could be static in I guess and missing types.
Also maybe `getScrollParent`, not `scrolled`?